03/3_is_array_overlap/main.c

```diff
@@ -0,0 +1,34 @@
+#include <stdio.h>
+
+int is_overlap(int nums1[], const size_t length1, int nums2[], const size_t length2);
+
+int main(void)
+{
+    int nums1[5] = {1, 2, 3, 4, 5};
+    int nums2[5] = {1, 2, 8, 9, 10};
+    int* nums3 = nums1;
+    const size_t LENGTH = 5u;

```

Comment:
if you are going to make this as const.. u should use this for array initialization as well..

if not sizeof(nums1) / sizeof(nums1[0]) is a better way 

or make a macro for this #define ARRAY_LENGTH(arr) (sizeof(arr) / sizeof(arr[0])